### ISSUE TITLE
Adding metadata to nodes in returned json objects

### ISSUE THREAD

**Issue by cmungall** [SciGraph/SciGraph#4]
It would be useful to have additional metadata on each node. The bbop tpology graph format here is quite open ended (@kltm check if this is correct):

```
  {id: ID,
   lbl: LABEL,
   meta: {
       KEY1: VAL1,
       ...
       KEYn: VALn
   },
```

Does this map well to the Neo4j model (with the meta object corresponding to node attributes in neo4j)? It would make things simplest to keep things aligned.

The key-val pairs would come originally from annotation assertion axioms in the ontology. (except "category" which is derived independently).

It would be useful to have category exposed so we can use it to color nodes etc

**Comment by ccondit** [comment]
Currently all node properties are returned with other graph formats but the bbop.js format includes just id, label, and category. Since the `meta` object in bbop.js may contain arbitrary properties would it make sense to unify the behavior of all these formats and add some mechanism to control the projection of the resulting graph with some sensible defaults (also allowing a `*` projection to return everything)?

Related to #115

/cc @nlwashington

**Comment by kltm** [comment]
What kind of properties are being discussed here? If it goes beyond key/value, it might be worth either subclassing or having the data available in a separate properties class.

**Comment by ccondit** [comment]
Is this case any annotation property or data property. I think it can effectively be considered key/value.

**Comment by ccondit** [comment]
This is resolved with #121.
